ci/pipelines/deploy-to-test.yml

```diff
@@ -1452,6 +1467,36 @@ jobs:
     <<: *put_success_slack_notification
     <<: *put_failure_slack_notification
 
+  - name: record-toolbox-release-number
+    serial_groups: [hosted-graphite]
+    plan:
+      - get: toolbox-ecr-registry-test
+        trigger: true
+        passed: [deploy-toolbox]
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: toolbox-ecr-registry-test/tag
+      - task: send-release-info-to-hosted-graphite
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: alpine
+          params:
+            ENVIRONMENT: "test-12"
+            APP_NAME: "toolbox"
+            RELEASE_NUMBER: ((.:application_image_tag))
+            HOSTED_GRAPHITE_API_TOKEN: ((pr-ci/HOSTED_GRAPHITE_API_KEY))
```

Comment:
We shouldn't reuse the token from another pipeline in this one, this being prefixed with pr-ci means the token is supposed to be scoped to the pr-ci pipeline (weirdly it feels like a bug that we can even get it!)

Reply:
Oh blimey I forgot I'd done this - good spot, will fix